litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/ColorMatrixFilter.java

```diff
@@ -0,0 +1,65 @@
+package com.linkedin.android.litr.filter.video.gl;
+
+import android.graphics.PointF;
+import android.opengl.GLES20;
+
+import androidx.annotation.NonNull;
+
+/**
+ * Created by tapos-datta on 9/14/20.
```

Comment:
Small request: can you add BSD licensing header in the beginning (other classes have it) and move this line to the end of it?

Reply:
Actually, if this is coming from `android-gpuimage` project, we should use their copyright notice

Reply:
Sure, I have updated with corresponding licenses into the source files.